distributed/src/test/scala/asobu/distributed/util/SerializableTest.scala

```diff
@@ -0,0 +1,27 @@
+package asobu.distributed.util
+
+import scala.util.control.NonFatal
+
+trait SerializableTest {
+  def isSerializable[T](a: T): Boolean = {
+    import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+    val baos = new ByteArrayOutputStream()
+    val oos = new ObjectOutputStream(baos)
+    var ois: ObjectInputStream = null
+    try {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using util.Try instead of a try-catch block](https://www.codacy.com/app/iHeartRadio/asobu/file/2296165730/issues/source?bid=3037259&fileBranchId=3178877#l11)